proto/ds/dsdata.proto

```diff
@@ -24,7 +24,7 @@ message HiddenDataKey {
   bytes ephemeralPubkey = 2;
 
   // SharedSecret = privkey * ephemeral_privkey * Generator = privkey * ephemeralPubkey
-  // encryptedKey = AES_ENCRYPT(key=sha256(SharedSecret)[:128], iv=iv, data=data_key)
+  // encryptedHiddenKey = AES_ENCRYPT(key=sha256(SharedSecret)[:128], iv=iv, data=data_key)
   bytes encryptedHiddenKey = 10; // AES_128 encryption
```

Comment:
Let's called this the `encryptedDataKey`, which is the encrypted version of the original `data_key`